scripts/caclmgrd

```diff
@@ -154,38 +154,59 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
             self.update_docker_mgmt_ip_acl(fabric_asic_namespace)
 
     def update_docker_mgmt_ip_acl(self, namespace):
-            self.iptables_cmd_ns_prefix[namespace] = "ip netns exec " + namespace + " "
+            self.iptables_cmd_ns_prefix[namespace] = ["ip", "netns", "exec", str(namespace)]
             self.namespace_docker_mgmt_ip[namespace] = self.get_namespace_mgmt_ip(self.iptables_cmd_ns_prefix[namespace],
                                                                                              namespace)
             self.namespace_docker_mgmt_ipv6[namespace] = self.get_namespace_mgmt_ipv6(self.iptables_cmd_ns_prefix[namespace],
                                                                                              namespace)
 
     def get_namespace_mgmt_ip(self, iptable_ns_cmd_prefix, namespace):
-        ip_address_get_command = iptable_ns_cmd_prefix + "ip -4 -o addr show " + ("eth0" if namespace else "docker0") +\
-                                 " | awk '{print $4}' | cut -d'/' -f1 | head -1"
+        ip_address_cmd0 = iptable_ns_cmd_prefix + ['ip', '-4', '-o', 'addr', 'show', ("eth0" if namespace else "docker0")]
+        ip_address_cmd1 = ['awk', '{print $4}']
+        ip_address_cmd2 = ['cut', '-d', '/', '-f1']
+        ip_address_cmd3 = ['head', '-1']
 
-        return self.run_commands([ip_address_get_command])
+        return self.run_commands_pipe(ip_address_cmd0, ip_address_cmd1, ip_address_cmd2, ip_address_cmd3)
 
     def get_namespace_mgmt_ipv6(self, iptable_ns_cmd_prefix, namespace):
-        ipv6_address_get_command = iptable_ns_cmd_prefix + "ip -6 -o addr show scope global " + ("eth0" if namespace else "docker0") +\
-                                 " | awk '{print $4}' | cut -d'/' -f1 | head -1"
-        return self.run_commands([ipv6_address_get_command])
+        ipv6_address_cmd0 = iptable_ns_cmd_prefix + ['ip', '-6', '-o', 'addr', 'show', 'scope', 'global', ("eth0" if namespace else "docker0")]
+        ipv6_address_cmd1 = ['awk', '{print $4}']
+        ipv6_address_cmd2 = ['cut', '-d', '/', '-f1']
+        ipv6_address_cmd3 = ['head', '-1']
+        return self.run_commands_pipe(ipv6_address_cmd0, ipv6_address_cmd1, ipv6_address_cmd2, ipv6_address_cmd3)
+
+    def log_output(self, cmd, exitcodes, stdout):
```

Comment:
>log_output [](http://example.com/codeflow?start=8&length=10)

Add an explicit `return None` at the function end, since you are expecting the behavior.